tcms/static/style/patternfly_override.css

```diff
@@ -39,3 +39,15 @@ nav.navbar-default { border-top-color: #3C8D2C; }
     border: 1px solid #ccc;
     padding: 5px;
 }
+
+@media print {
+    a[href]:after,
+    form.toolbar-pf-actions,
+    .card-pf-accented tfoot,
+    .list-view-pf-actions,
+    .list-view-pf-checkbox,
+    .pficon-error-circle-o,
+    .tree-list-view-pf .comment-form {
+        display: none !important;
+    }
```

Comment:
Patternfly already has the `.hidden-print` class definition and we'd like to use that as much as possible.

Also we want to have elements in  HTML templates explicitly marked with this class to that it is clear they are not intended to be printed on the screen. 

An extension for the `:after` elements showing the URLs will be needed probably.